docs/azure/sdk/authentication/index.md

```diff
@@ -76,6 +76,13 @@ A service principal is created in a Microsoft Entra tenant to represent an app a
 > [!div class="nextstepaction"]
 > [Authenticate locally using a service principal](local-development-service-principal.md)
 
+#### Use a broker
```

Comment:
Move this section ahead of the "Use a service principal" section, as it's better than that approach